forge.config.ts

```diff
@@ -9,23 +9,8 @@ const rootDir = process.cwd()
 const config: ForgeConfig = {
   packagerConfig: {
     asar: true,
-    osxSign: (process.env.BUILD_RELEASE === 'true' && {}) || undefined,
-    osxNotarize:
-      (process.env.BUILD_RELEASE === 'true' && {
-        appleId: process.env.APPLE_ID || '',
-        appleIdPassword: process.env.APPLE_PASSWORD || '',
-        teamId: process.env.APPLE_TEAM_ID || '',
-      }) ||
-      undefined,
     executableName: 'zoo-modeling-app',
     icon: path.resolve(rootDir, 'assets', 'icon'),
-    protocols: [
-      {
-        name: 'Zoo Studio',
-        schemes: ['zoo-studio'],
-      },
-    ],
-    extendInfo: 'Info.plist', // Information for file associations.
```

Comment:
Moved this to electron-builder.yml, that we're using to package the app. This was added before the migration to builder.